### PR TITLE
Chore: spelling - ui

### PR DESCRIPTION
**Pull request by jsoref**
**What this PR does / why we need it**:

This fixes a number of spelling errors.

**Special notes for your reviewer**:

Comments on file changes can be found in #24412

### CODE REVIEW

File: packages/grafana-ui/src/components/Graph/GraphLegend.story.tsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { LegendPlacement, LegendDisplayMode } from '../Legend/Legend';
 export default {
   title: 'Visualizations/Graph/GraphLegend',
   component: GraphLegend,
-  decororators: [withHorizontallyCenteredStory],
+  decorators: [withHorizontallyCenteredStory],
```
Comment (marcusolsson) [inline packages/grafana-ui/src/components/Graph/GraphLegend.story.tsx:12]: @tskarhed Is this a breaking change?

File: packages/grafana-ui/src/components/Legend/Legend.story.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const getStoriesKnobs = (table = false) => {
 
   const typeSpecificRenderer = table
     ? {
-        'Custom renderer(GraphLegendTablerow)': 'custom-tabe',
+        'Custom renderer(GraphLegendTablerow)': 'custom-table',
```
Comment (marcusolsson) [inline packages/grafana-ui/src/components/Legend/Legend.story.tsx:36]: @tskarhed Is this a breaking change?

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts [hunk 3]
```diff
@@ -377,12 +377,12 @@ $panel-grid-placeholder-bg: lighten(${theme.palette.blue95}, 30%);
 $panel-grid-placeholder-shadow: 0 0 4px ${theme.palette.blue95};
 
 // logs
-$logs-color-unkown: $gray-5;
+$logs-color-unknown: $gray-5;
```
Comment (marcusolsson) [inline packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:380]: Need to make sure we update references to this.

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts [hunk 4]
```diff
@@ -377,12 +377,12 @@ $panel-grid-placeholder-bg: lighten(${theme.palette.blue95}, 30%);
...
 
 // toggle-group
 $button-toggle-group-btn-active-bg: $brand-primary;
 $button-toggle-group-btn-active-shadow: inset 0 0 4px $white;
-$button-toggle-group-btn-seperator-border: 1px solid $gray-6;
+$button-toggle-group-btn-separator-border: 1px solid $gray-6;
```
Comment (marcusolsson) [inline packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:385]: Need to make sure we update references to this.

File: packages/grafana-ui/src/components/Graph/GraphLegend.story.tsx (hunk 1 above)
Comment (tskarhed) [reply packages/grafana-ui/src/components/Graph/GraphLegend.story.tsx:12]: It's actually a fixing change! :smile:

File: packages/grafana-ui/src/components/Legend/Legend.story.tsx (hunk 2 above)
Comment (tskarhed) [reply packages/grafana-ui/src/components/Legend/Legend.story.tsx:36]: This seems to work

**Review by jessover9000 — APPROVED** [review]
Thanks for fixing those!

**Comment by jsoref** [comment]
@jessover9000 ?

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts (hunk 4 above)
Comment (tskarhed) [reply packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:385]: This one is used in _toggle_button_group.scss

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts (hunk 3 above)
Comment (tskarhed) [reply packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:380]: Couldn't find a place where this is used. It should probably be removed.

File: packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts (hunk 4 above)
Comment (tskarhed) [reply packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:385]: @jsoref Is this updated in another PR?

Comment (tskarhed) [reply packages/grafana-ui/src/themes/_variables.light.scss.tmpl.ts:385]: Yeah, this change in another PR broke the build during Friday, so it should be okay now.

**Comment by jsoref** [comment]
:-(

**Pull request merged**